src/test/java/GetSasToken.java

```diff
@@ -0,0 +1,39 @@
+import java.net.URLEncoder;
+import java.util.Base64;
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class GetSasToken {
```

Comment:
Don't  need this class duplicate ?

Reply:
its a bit odd since I thought I removed it but its still around ... I think something is a bit screwy at the moment Ill investigate.